examples/architectureComponentsExample/src/main/java/io/realm/examples/arch/MyApplication.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.arch;
+
+import android.app.Application;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+import io.realm.examples.arch.model.Person;
+
+
+public class MyApplication extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        Realm.init(this);
+        final RealmConfiguration defaultConfig = new RealmConfiguration.Builder().initialData(new Realm.Transaction() {
```

Comment:
I don't understand this `final`.  Why introduce an un-necessary variable, but make if final?